src/overlays/actors/ovl_En_Minislime/z_en_minislime.h

```diff
@@ -5,9 +5,49 @@
 
 struct EnMinislime;
 
+typedef void (*EnMinislimeActionFunc)(struct EnMinislime*, GlobalContext*);
+
+#define MINISLIME_NUM_SPAWN 15
```

Comment:
Perhaps this is a better name, but I'm happy with the one in the PR.
```suggestion
#define MINISLIME_SPLIT_COUNT 15
```